backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryHttpClient.java

```diff
@@ -166,14 +166,17 @@ private TelemetryMeasuresPayload createMeasuresPayload(TelemetryLocalStorage dat
       ))
       .forEach(values::add);
 
-    data.getQuickFixCountByRuleKey().entrySet().stream()
-      .map(e -> {
-        var ruleKey = e.getKey();
-        var ruleRepoAndId = ruleKey.split(":");
-        var measureKey = String.format("quick_fix.applied_count.%s.%s", ruleRepoAndId[0], ruleRepoAndId[1]);
-        return new TelemetryMeasuresValue(measureKey, String.valueOf(e.getValue()), INTEGER, DAILY);
-      })
-      .forEach(values::add);
+    var allQuickFixCount = data.getQuickFixCountByRuleKey().values().stream()
+      .mapToInt(Integer::intValue)
+      .sum();
+    if (allQuickFixCount > 0) {
```

Comment:
Don't we also want to collect the measure if the count is 0?

Reply:
As far as I can tell, the goal is to count quick fixes applied on a given period, so reporting 0 as a quick fix count for a given period has the same effect as not reporting a value, with an additional row in the generated measures table. My guess is that this will be the case for most installs, so a lot of not-so-useful lines with a value of 0 in the table.